src/com/sahokia/kos/bustickets/BusTicketsApplication.java

```diff
@@ -0,0 +1,12 @@
+package com.sahokia.kos.bustickets;
+
+import com.sahokia.kos.bustickets.service.BusTicketsService;
+
+public class BusTicketsApplication {
+    public static void main(String[] args) {
+        BusTicketsService busTicketsService = new BusTicketsService();
+        String filePath = "src/com/sahokia/kos/bustickets/data/BusTickets.txt";
+        busTicketsService.mapTicketsFromFile(filePath);
```

Comment:
Now the ticketService methods are able to work with input lists, and also return lists from mapping as a result of method, so we can save them into a varibale. In this case I have removed inner storage of tickets, because in general we don't really need it anymore. Done.